pkg/controller/repository/repository_controller_test.go

```diff
@@ -42,7 +42,13 @@ const (
 var (
 	_ reconcile.Reconciler = &ReconcileRepository{}
 
-	testResources = []pipelinev1beta1.PipelineResourceBinding{{Name: "testing"}}
+	testResources  = []pipelinev1beta1.PipelineResourceBinding{{Name: "testing"}}
```

Comment:
Can  you maybe gofmt this?

Reply:
Thank you for reviewing!

I have already used gofmt.
Should I remove a whitespace before ```=```?

Reply:
@hnts please do, I assumed that gofmt would fix that :-)

After that, I'm happy to merge this.

Reply:
I fixed it!